SignInActivity does not have a NavController

```
[SUMMARY]
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.flowcrypt.email/com.flowcrypt.email.ui.activity.SignInActivity}: java.lang.IllegalStateException: Activity com.flowcrypt.email.ui.activity.SignInActivity@85dd924 does not have a NavController set on 2131296568
java.lang.IllegalStateException: Activity com.flowcrypt.email.ui.activity.SignInActivity@85dd924 does not have a NavController set on 2131296568


[STACK_TRACE]
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.flowcrypt.email/com.flowcrypt.email.ui.activity.SignInActivity}: java.lang.IllegalStateException: Activity com.flowcrypt.email.ui.activity.SignInActivity@85dd924 does not have a NavController set on 2131296568
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3449)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3601)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2066)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
Caused by: java.lang.IllegalStateException: Activity com.flowcrypt.email.ui.activity.SignInActivity@85dd924 does not have a NavController set on 2131296568
        at androidx.navigation.Navigation.findNavController(Navigation.java:61)
        at androidx.navigation.ActivityKt.findNavController(Activity.kt:30)
        at com.flowcrypt.email.ui.activity.SignInActivity.onCreate(SignInActivity.kt:57)
        at android.app.Activity.performCreate(Activity.java:8000)
        at android.app.Activity.performCreate(Activity.java:7984)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1309)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3422)
        ... 11 more
```

--- a/FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/activity/ImportPgpContactActivityTest.kt
+++ b/FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/activity/ImportPgpContactActivityTest.kt
@@ -45,9 +45,8 @@ import org.hamcrest.CoreMatchers.containsString
 import org.hamcrest.CoreMatchers.equalTo
 import org.hamcrest.CoreMatchers.hasItem
 import org.hamcrest.CoreMatchers.not
-import org.junit.AfterClass
-import org.junit.BeforeClass
-import org.junit.ClassRule
+import org.junit.After
+import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
 import org.junit.rules.RuleChain
@@ -75,13 +74,62 @@ class ImportPgpContactActivityTest : BaseTest() {
     )
   )
 
+  private lateinit var fileWithPublicKey: File
+  private lateinit var publicKey: String
+
+  private val mockWebServerRule = FlowCryptMockWebServerRule(TestConstants.MOCK_WEB_SERVER_PORT,
+    object : Dispatcher() {
+      override fun dispatch(request: RecordedRequest): MockResponse {
+        if (request.path?.startsWith("/pub", ignoreCase = true) == true) {
+          val lastSegment = request.requestUrl?.pathSegments?.lastOrNull()
+
+          when {
+            TestConstants.RECIPIENT_WITHOUT_PUBLIC_KEY_ON_ATTESTER.equals(
+              lastSegment, true
+            ) -> {
+              return MockResponse()
+                .setStatus("HTTP/1.1 404 Not Found")
+                .setBody(TestGeneralUtil.readResourceAsString("2.txt"))
+            }
+
+            TestConstants.RECIPIENT_WITH_PUBLIC_KEY_ON_ATTESTER.equals(
+              lastSegment, true
+            ) -> {
+              return MockResponse()
+                .setResponseCode(HttpURLConnection.HTTP_OK)
+                .setBody(TestGeneralUtil.readResourceAsString("3.txt"))
+            }
+          }
+        }
+
+        return MockResponse().setResponseCode(HttpURLConnection.HTTP_NOT_FOUND)
+      }
+    })
+
   @get:Rule
   var ruleChain: TestRule = RuleChain
     .outerRule(ClearAppSettingsRule())
+    .around(mockWebServerRule)
     .around(addAccountToDatabaseRule)
     .around(RetryRule.DEFAULT)
     .around(activityScenarioRule)
     .around(ScreenshotTestRule())
+
+  @Before
+  fun createResources() {
+    publicKey = TestGeneralUtil.readFileFromAssetsAsString(
+      "pgp/" + TestConstants.RECIPIENT_WITHOUT_PUBLIC_KEY_ON_ATTESTER + "-pub.asc"
+    )
+    fileWithPublicKey = TestGeneralUtil.createFileAndFillWithContent(
+      fileName = TestConstants.RECIPIENT_WITHOUT_PUBLIC_KEY_ON_ATTESTER + "_pub.asc",
+      fileText = publicKey
+    )
+  }
+
+  @After
+  fun cleanResources() {
+    TestGeneralUtil.deleteFiles(listOf(fileWithPublicKey))
+  }
 
   @Test
   fun testFetchKeyFromAttesterForExistedUser() {
@@ -164,59 +212,5 @@ class ImportPgpContactActivityTest : BaseTest() {
       .perform(click())
     onView(withText(containsString(TestConstants.RECIPIENT_WITHOUT_PUBLIC_KEY_ON_ATTESTER)))
       .check(matches(isDisplayed()))
-  }
-
-  companion object {
-    private lateinit var fileWithPublicKey: File
-    private lateinit var publicKey: String
-
-    @get:ClassRule
-    @JvmStatic
-    val mockWebServerRule = FlowCryptMockWebServerRule(TestConstants.MOCK_WEB_SERVER_PORT,
-      object : Dispatcher() {
-        override fun dispatch(request: RecordedRequest): MockResponse {
-          if (request.path?.startsWith("/pub", ignoreCase = true) == true) {
-            val lastSegment = request.requestUrl?.pathSegments?.lastOrNull()
-
-            when {
-              TestConstants.RECIPIENT_WITHOUT_PUBLIC_KEY_ON_ATTESTER.equals(
-                lastSegment, true
-              ) -> {
-                return MockResponse()
-                  .setStatus("HTTP/1.1 404 Not Found")
-                  .setBody(TestGeneralUtil.readResourceAsString("2.txt"))
-              }
-
-              TestConstants.RECIPIENT_WITH_PUBLIC_KEY_ON_ATTESTER.equals(
-                lastSegment, true
-              ) -> {
-                return MockResponse()
-                  .setResponseCode(HttpURLConnection.HTTP_OK)
-                  .setBody(TestGeneralUtil.readResourceAsString("3.txt"))
-              }
-            }
-          }
-
-          return MockResponse().setResponseCode(HttpURLConnection.HTTP_NOT_FOUND)
-        }
-      })
-
-    @BeforeClass
-    @JvmStatic
-    fun createResources() {
-      publicKey = TestGeneralUtil.readFileFromAssetsAsString(
-        "pgp/" + TestConstants.RECIPIENT_WITHOUT_PUBLIC_KEY_ON_ATTESTER + "-pub.asc"
-      )
-      fileWithPublicKey = TestGeneralUtil.createFileAndFillWithContent(
-        fileName = TestConstants.RECIPIENT_WITHOUT_PUBLIC_KEY_ON_ATTESTER + "_pub.asc",
-        fileText = publicKey
-      )
-    }
-
-    @AfterClass
-    @JvmStatic
-    fun cleanResources() {
-      TestGeneralUtil.deleteFiles(listOf(fileWithPublicKey))
-    }
   }
 }

--- a/FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/activity/ShareIntentsTest.kt
+++ b/FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/activity/ShareIntentsTest.kt
@@ -36,9 +36,8 @@ import okhttp3.mockwebserver.RecordedRequest
 import org.hamcrest.Matchers.containsString
 import org.hamcrest.Matchers.isEmptyString
 import org.hamcrest.Matchers.not
-import org.junit.AfterClass
-import org.junit.BeforeClass
-import org.junit.ClassRule
+import org.junit.After
+import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
 import org.junit.rules.RuleChain
@@ -67,9 +66,33 @@ class ShareIntentsTest : BaseTest() {
   override val activityScenario: ActivityScenario<*>?
     get() = activeActivityRule.scenario
 
+  private val mockWebServerRule =
+    FlowCryptMockWebServerRule(TestConstants.MOCK_WEB_SERVER_PORT, object : Dispatcher() {
+      override fun dispatch(request: RecordedRequest): MockResponse {
+        if (request.path?.startsWith("/pub", ignoreCase = true) == true) {
+          val lastSegment = request.requestUrl?.pathSegments?.lastOrNull()
+
+          when {
+            TestConstants.RECIPIENT_WITHOUT_PUBLIC_KEY_ON_ATTESTER.equals(lastSegment, true) -> {
+              return MockResponse().setResponseCode(HttpURLConnection.HTTP_NOT_FOUND)
+                .setBody(TestGeneralUtil.readResourceAsString("2.txt"))
+            }
+
+            TestConstants.RECIPIENT_WITH_PUBLIC_KEY_ON_ATTESTER.equals(lastSegment, true) -> {
+              return MockResponse().setResponseCode(HttpURLConnection.HTTP_OK)
+                .setBody(TestGeneralUtil.readResourceAsString("3.txt"))
+            }
+          }
+        }
+
+        return MockResponse().setResponseCode(HttpURLConnection.HTTP_NOT_FOUND)
+      }
+    })
+
   @get:Rule
   var ruleChain: TestRule = RuleChain
     .outerRule(ClearAppSettingsRule())
+    .around(mockWebServerRule)
     .around(AddAccountToDatabaseRule())
     .around(RetryRule.DEFAULT)
     .around(activeActivityRule)
@@ -77,6 +100,27 @@ class ShareIntentsTest : BaseTest() {
 
   private val randomActionForRFC6068: String
     get() = if (Random().nextBoolean()) Intent.ACTION_SENDTO else Intent.ACTION_VIEW
+
+  private lateinit var atts: MutableList<File>
+
+  @Before
+  fun setUp() {
+    atts = mutableListOf()
+
+    for (i in 0 until ATTACHMENTS_COUNT) {
+      atts.add(
+        TestGeneralUtil.createFileAndFillWithContent(
+          "$i.txt",
+          UUID.randomUUID().toString()
+        )
+      )
+    }
+  }
+
+  @After
+  fun cleanResources() {
+    TestGeneralUtil.deleteFiles(atts)
+  }
 
   @Test
   fun testEmptyUri() {
@@ -424,60 +468,9 @@ class ShareIntentsTest : BaseTest() {
     private const val ENCODED_SUBJECT = "some%20subject"
     private const val ENCODED_BODY = "some%20body"
 
-    private lateinit var atts: MutableList<File>
     private val recipients: Array<String> = arrayOf(
       TestConstants.RECIPIENT_WITH_PUBLIC_KEY_ON_ATTESTER,
       TestConstants.RECIPIENT_WITHOUT_PUBLIC_KEY_ON_ATTESTER
     )
-
-    @BeforeClass
-    @JvmStatic
-    fun setUp() {
-      createFilesForAtts()
-    }
-
-    @AfterClass
-    @JvmStatic
-    fun cleanResources() {
-      TestGeneralUtil.deleteFiles(atts)
-    }
-
-    @get:ClassRule
-    @JvmStatic
-    val mockWebServerRule =
-      FlowCryptMockWebServerRule(TestConstants.MOCK_WEB_SERVER_PORT, object : Dispatcher() {
-        override fun dispatch(request: RecordedRequest): MockResponse {
-          if (request.path?.startsWith("/pub", ignoreCase = true) == true) {
-            val lastSegment = request.requestUrl?.pathSegments?.lastOrNull()
-
-            when {
-              TestConstants.RECIPIENT_WITHOUT_PUBLIC_KEY_ON_ATTESTER.equals(lastSegment, true) -> {
-                return MockResponse().setResponseCode(HttpURLConnection.HTTP_NOT_FOUND)
-                  .setBody(TestGeneralUtil.readResourceAsString("2.txt"))
-              }
-
-              TestConstants.RECIPIENT_WITH_PUBLIC_KEY_ON_ATTESTER.equals(lastSegment, true) -> {
-                return MockResponse().setResponseCode(HttpURLConnection.HTTP_OK)
-                  .setBody(TestGeneralUtil.readResourceAsString("3.txt"))
-              }
-            }
-          }
-
-          return MockResponse().setResponseCode(HttpURLConnection.HTTP_NOT_FOUND)
-        }
-      })
-
-    private fun createFilesForAtts() {
-      atts = mutableListOf()
-
-      for (i in 0 until ATTACHMENTS_COUNT) {
-        atts.add(
-          TestGeneralUtil.createFileAndFillWithContent(
-            "$i.txt",
-            UUID.randomUUID().toString()
-          )
-        )
-      }
-    }
   }
 }

--- a/FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/activity/enterprise/CreatePrivateKeyActivityEnterpriseTest.kt
+++ b/FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/activity/enterprise/CreatePrivateKeyActivityEnterpriseTest.kt
@@ -35,7 +35,6 @@ import com.flowcrypt.email.util.AccountDaoManager
 import okhttp3.mockwebserver.Dispatcher
 import okhttp3.mockwebserver.MockResponse
 import okhttp3.mockwebserver.RecordedRequest
-import org.junit.ClassRule
 import org.junit.Rule
 import org.junit.Test
 import org.junit.rules.RuleChain
@@ -62,9 +61,33 @@ class CreatePrivateKeyActivityEnterpriseTest : BasePassphraseActivityTest() {
       )
     })
 
+  private val mockWebServerRule =
+    FlowCryptMockWebServerRule(TestConstants.MOCK_WEB_SERVER_PORT, object : Dispatcher() {
+      override fun dispatch(request: RecordedRequest): MockResponse {
+        val gson =
+          ApiHelper.getInstance(InstrumentationRegistry.getInstrumentation().targetContext).gson
+        val model = gson.fromJson(
+          InputStreamReader(request.body.inputStream()),
+          InitialLegacySubmitModel::class.java
+        )
+
+        if (request.path.equals("/initial/legacy_submit")) {
+          when (model.email) {
+            EMAIL_ENFORCE_ATTESTER_SUBMIT -> return MockResponse().setResponseCode(
+              HttpURLConnection.HTTP_OK
+            )
+              .setBody(gson.toJson(SUBMIT_API_ERROR_RESPONSE))
+          }
+        }
+
+        return MockResponse().setResponseCode(HttpURLConnection.HTTP_NOT_FOUND)
+      }
+    })
+
   @get:Rule
   var ruleChain: TestRule = RuleChain
     .outerRule(ClearAppSettingsRule())
+    .around(mockWebServerRule)
     .around(RetryRule.DEFAULT)
     .around(activityScenarioRule)
     .around(ScreenshotTestRule())
@@ -100,30 +123,5 @@ class CreatePrivateKeyActivityEnterpriseTest : BasePassphraseActivityTest() {
             "address", "internal_error"
       ), false
     )
-
-    @get:ClassRule
-    @JvmStatic
-    val mockWebServerRule =
-      FlowCryptMockWebServerRule(TestConstants.MOCK_WEB_SERVER_PORT, object : Dispatcher() {
-        override fun dispatch(request: RecordedRequest): MockResponse {
-          val gson =
-            ApiHelper.getInstance(InstrumentationRegistry.getInstrumentation().targetContext).gson
-          val model = gson.fromJson<InitialLegacySubmitModel>(
-            InputStreamReader(request.body.inputStream()),
-            InitialLegacySubmitModel::class.java
-          )
-
-          if (request.path.equals("/initial/legacy_submit")) {
-            when (model.email) {
-              EMAIL_ENFORCE_ATTESTER_SUBMIT -> return MockResponse().setResponseCode(
-                HttpURLConnection.HTTP_OK
-              )
-                .setBody(gson.toJson(SUBMIT_API_ERROR_RESPONSE))
-            }
-          }
-
-          return MockResponse().setResponseCode(HttpURLConnection.HTTP_NOT_FOUND)
-        }
-      })
   }
 }

--- a/FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/SignInActivity.kt
+++ b/FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/SignInActivity.kt
@@ -11,10 +11,8 @@ import android.app.Activity
 import android.content.Intent
 import android.os.Bundle
 import android.view.View
-import androidx.navigation.findNavController
 import com.flowcrypt.email.BuildConfig
 import com.flowcrypt.email.R
-import com.flowcrypt.email.SingInGraphDirections
 import com.flowcrypt.email.ui.activity.base.BaseActivity
 import com.flowcrypt.email.ui.activity.fragment.AddOtherAccountFragment
 import com.flowcrypt.email.ui.activity.fragment.UserRecoverableAuthExceptionFragment
@@ -50,15 +48,6 @@ class SignInActivity : BaseActivity() {
     accountAuthenticatorResponse =
       intent.getParcelableExtra(AccountManager.KEY_ACCOUNT_AUTHENTICATOR_RESPONSE)
     accountAuthenticatorResponse?.onRequestContinued()
-
-    if (savedInstanceState == null) {
-      when (intent.action) {
-        ACTION_UPDATE_OAUTH_ACCOUNT -> {
-          findNavController(R.id.fragmentContainerView)
-            .navigate(SingInGraphDirections.actionGlobalUserRecoverableAuthExceptionFragment())
-        }
-      }
-    }
   }
 
   override fun onDestroy() {
@@ -83,20 +72,11 @@ class SignInActivity : BaseActivity() {
     super.onNewIntent(intent)
     val fragments =
       supportFragmentManager.primaryNavigationFragment?.childFragmentManager?.fragments
-    val fragment = when (intent?.action) {
-      ACTION_UPDATE_OAUTH_ACCOUNT -> {
-        fragments?.firstOrNull {
-          it is UserRecoverableAuthExceptionFragment
-        }
-      }
-
-      else -> {
-        fragments?.firstOrNull {
-          it is AddOtherAccountFragment
-        }
-      }
+    val fragment = fragments?.firstOrNull {
+      it is UserRecoverableAuthExceptionFragment
+    } ?: fragments?.firstOrNull {
+      it is AddOtherAccountFragment
     }
-
     val oAuthFragment = fragment as? BaseOAuthFragment
     oAuthFragment?.handleOAuth2Intent(intent)
   }
@@ -106,8 +86,6 @@ class SignInActivity : BaseActivity() {
       BuildConfig.APPLICATION_ID + ".ACTION_ADD_ONE_MORE_ACCOUNT"
     const val ACTION_ADD_ACCOUNT_VIA_SYSTEM_SETTINGS =
       BuildConfig.APPLICATION_ID + ".ACTION_ADD_ACCOUNT_VIA_SYSTEM_SETTINGS"
-    const val ACTION_UPDATE_OAUTH_ACCOUNT =
-      BuildConfig.APPLICATION_ID + ".ACTION_UPDATE_OAUTH_ACCOUNT"
 
     val KEY_EXTRA_NEW_ACCOUNT =
       GeneralUtil.generateUniqueExtraKey("KEY_EXTRA_NEW_ACCOUNT", SignInActivity::class.java)

--- a/FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/UserRecoverableAuthExceptionFragment.kt
+++ b/FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/UserRecoverableAuthExceptionFragment.kt
@@ -194,7 +194,7 @@ class UserRecoverableAuthExceptionFragment : BaseOAuthFragment(), ProgressBehavi
                     requireContext(), 0, Intent(
                       requireContext(),
                       SignInActivity::class.java
-                    ).apply { action = SignInActivity.ACTION_UPDATE_OAUTH_ACCOUNT }, 0
+                    ), 0
                   )
                 )
             }

--- a/FlowCrypt/src/main/java/com/flowcrypt/email/ui/notifications/ErrorNotificationManager.kt
+++ b/FlowCrypt/src/main/java/com/flowcrypt/email/ui/notifications/ErrorNotificationManager.kt
@@ -11,13 +11,13 @@ import android.content.Context
 import android.content.Intent
 import android.graphics.BitmapFactory
 import android.net.Uri
+import android.os.Bundle
 import androidx.core.app.NotificationCompat
 import com.flowcrypt.email.BuildConfig
 import com.flowcrypt.email.R
 import com.flowcrypt.email.database.FlowCryptRoomDatabase
 import com.flowcrypt.email.database.entity.AccountEntity
 import com.flowcrypt.email.ui.activity.EmailManagerActivity
-import com.flowcrypt.email.ui.activity.SignInActivity
 
 /**
  * It's a manager which helps to show errors notifications with a high priority.
@@ -141,9 +141,14 @@ class ErrorNotificationManager(context: Context) : CustomNotificationManager(con
     ): Intent {
       return when (account?.accountType) {
         AccountEntity.ACCOUNT_TYPE_GOOGLE, AccountEntity.ACCOUNT_TYPE_OUTLOOK -> {
-          Intent(context, SignInActivity::class.java).apply {
-            action = SignInActivity.ACTION_UPDATE_OAUTH_ACCOUNT
-            recoverableIntent?.let { putExtra("recoverableIntent", recoverableIntent) }
+          val uri = Uri.parse("flowcrypt://email.flowcrypt.com/sign-in/recover_auth")
+          Intent(null, uri).apply {
+            recoverableIntent?.let {
+              val extras = Bundle().apply {
+                putParcelable("recoverableIntent", recoverableIntent)
+              }
+              putExtra("android-support-nav:controller:deepLinkExtras", extras)
+            }
           }
         }
 

--- a/FlowCrypt/src/main/res/navigation/nav_graph.xml
+++ b/FlowCrypt/src/main/res/navigation/nav_graph.xml
@@ -247,18 +247,6 @@
     </fragment>
 
     <fragment
-        android:id="@+id/userRecoverableAuthExceptionFragment"
-        android:name="com.flowcrypt.email.ui.activity.fragment.UserRecoverableAuthExceptionFragment"
-        android:label="UserRecoverableAuthExceptionFragment"
-        tools:layout="@layout/fragment_user_recoverable_auth_exception">
-        <argument
-            android:name="recoverableIntent"
-            android:defaultValue="@null"
-            app:argType="android.content.Intent"
-            app:nullable="true" />
-    </fragment>
-
-    <fragment
         android:id="@+id/searchBackupsInEmailFragment"
         android:name="com.flowcrypt.email.ui.activity.fragment.SearchBackupsInEmailFragment"
         tools:layout="@layout/fragment_search_backups_in_email">

--- a/FlowCrypt/src/main/res/navigation/sing_in_graph.xml
+++ b/FlowCrypt/src/main/res/navigation/sing_in_graph.xml
@@ -52,8 +52,16 @@
         android:id="@+id/userRecoverableAuthExceptionFragment"
         android:name="com.flowcrypt.email.ui.activity.fragment.UserRecoverableAuthExceptionFragment"
         android:label="UserRecoverableAuthExceptionFragment"
-        tools:layout="@layout/fragment_user_recoverable_auth_exception" />
-
+        tools:layout="@layout/fragment_user_recoverable_auth_exception">
+        <deepLink
+            android:id="@+id/deepLink"
+            app:uri="flowcrypt://email.flowcrypt.com/sign-in/recover_auth" />
+        <argument
+            android:name="recoverableIntent"
+            android:defaultValue="@null"
+            app:argType="android.content.Intent"
+            app:nullable="true" />
+    </fragment>
     <fragment
         android:id="@+id/checkPassphraseStrengthFragment"
         android:name="com.flowcrypt.email.ui.activity.fragment.CheckPassphraseStrengthFragment"

--- a/script/ci-setup-and-run-emulator.sh
+++ b/script/ci-setup-and-run-emulator.sh
@@ -3,7 +3,7 @@
 "$ANDROID_SDK_ROOT/emulator/emulator" -accel-check
 echo -ne '\n' | avdmanager -v create avd --name ci-emulator --package "system-images;android-30;google_apis;x86_64" --device 'pixel_xl' --abi 'google_apis/x86_64'
 cat ~/.android/avd/ci-emulator.avd/config.ini
-echo "hw.ramSize=3064"  >> ~/.android/avd/ci-emulator.avd/config.ini
-cat ~/.android/avd/ci-emulator.avd/config.ini
+# echo "hw.ramSize=3064"  >> ~/.android/avd/ci-emulator.avd/config.ini
+# cat ~/.android/avd/ci-emulator.avd/config.ini
 "$ANDROID_SDK_ROOT/emulator/emulator" -list-avds #debug
 "$ANDROID_SDK_ROOT/emulator/emulator" -avd ci-emulator -no-window -no-boot-anim -no-audio &

